test/quantization/test_quantize_script.py

```diff
@@ -950,7 +953,7 @@ def forward(self, x):
                    .check("aten::adaptive_avg_pool2d") \
                    .check("dequantize") \
                    .run(m.graph)
-        res = get_forward(m._c)(x)
+        res = m(input)
```

Comment:
Wouldnt the listunpack op be needed prior to the maxpool and avgpool even after the swap dequantize pass?

Reply:
yeah, it's still there. I forgot to add the check for that